### ISSUE TITLE
consistency change: name field

### ISSUE THREAD

**Issue by Robbilie** [ccpgames/esi-issues#62]
alliance endpoint -> alliance_name
corporation endpoint -> corporation_name
character endpoint -> name

would be col if all three could be called 'name', alternatively ename the character endpoint. id prefer the former though :)

**Comment by a-tal** [comment]
@Robbilie mind listing the endpoints specifically? maybe in a task list <3

**Comment by Robbilie** [comment]
brb looking through all endpoints, what style is the one you would want?

**Comment by a-tal** [comment]
similar to https://github.com/ccpgames/esi-issues/issues/60

I think if any numbered list exists in the original comment it's used to track progress

**Comment by Robbilie** [comment]
the style question was wether you want to have all with type prefixed or all without :D

**Comment by a-tal** [comment]
I'm not a fan of redundancy, so probably the shorter one.

am a huge fan of consistency though, so if that's the case please do point them out

**Comment by Robbilie** [comment]
have fun with this giant list, i hope my destructuring syntax makes sense and i didn't fuck up

**Comment by polarina** [comment]
I think we would like to keep the prefix on the IDs as it helps disambiguate what is being referred to.

**Comment by aquarhead** [comment]
> killmails: why include id and not hash…?

1. The only way you can see a `killmail` resource, is to provide both its id and the correct "hash" yourself, so you already know the hash when accessing this killmail
2. This "hash" is not a property of the actual killmail
3. "hash" is more like an access control (or validation?) mechanism, ultimately locating a killmail relies on its id

**Comment by Robbilie** [comment]
mehr would be cool to have it in there anyways, will add it to my stored killmails anyways…

**Comment by Regner** [comment]
Came to add issue for `corporation_name` being inconsistent, it should just be `name`.

**Comment by Robbilie** [comment]
same for `alliance_name`

**Comment by Robbilie** [comment]
also the wars endpoint includes the war_id in the `id` field which is majorly inconsistent to be even existant

**Comment by polarina** [comment]
The ID field is always written out in full, i.e. `war_id`, `character_id`, etc.

**Comment by Robbilie** [comment]
no, the id field is inexistant on the characters, corporations and alliances endpoints

**Comment by polarina** [comment]
My apologies. I thought you were referring to a field that was named `war_id` and that you wanted it changed to `id`.

**Comment by Robbilie** [comment]
same goes for the types endpoint, i am happy with having the id field on the pages but if its there then please everywhere including char/corp/alli

**Comment by JimmyRoth** [comment]
Still needing to be renamed from system_id to solar_system_id:
- [ ] GET /universe/moons/{moon_id}/ 
- [ ] GET /universe/planets/{planet_id}/
- [ ] GET /universe/stations/{station_id}/
- [ ] GET /universe/stargates/{stargate_id}/

Note: stargates/{stargate_id}/ has ['destination'] system_id as well as the standard system_id

**Comment by Robbilie** [comment]
^ uhm it should be renamed TO system_id, not solar_system_id :)

**Comment by Blacksmoke16** [comment]
This is still a thing.  Are some endpoints using `system_id` and others using `solar_system_id`.  Should settle on which convention is going to be used to make life easier.

**Comment by lukasni** [comment]
Found another instance of `solar_system_id` -> `system_id` that wasn't caught in the changes from Dec 20th.

- [ ] GET /universe/stars/{star_id}/

Currently has `solar_system_id`, should be `system_id`

**Comment by lukasni** [comment]
-[] GET /v1/fw/systems/ `solar_system_id` to `system_id`
-[] GET /v1/fleets/{fleet_id}/members/  `solar_system_id` to `system_id`
-[] GET /v1/incursions/  `staging_solar_system_id` to `staging_system_id`
-[] GET /v1/industry/facilities/ `solar_system_id` to `system_id`
-[] GET /v1/industry/systems/ `solar_system_id` to `system_id`
-[] GET /v1/characters/{character_id}/mining/ `solar_system_id` to `system_id`
-[] GET /v1/killmails/{killmail_id}/{killmail_hash}/ `solar_system_id` to `system_id`
-[] GET /v1/characters/{character_id}/location/ `solar_system_id` to `system_id`
-[] GET /v1/characters/{character_id}/planets/ `solar_system_id` to `system_id`
-[] GET /v1/sovereignty/structures/ `solar_system_id` to `system_id`
-[] GET /v1/sovereignty/campaigns/ `solar_system_id` to `system_id`
-[] GET /v1/universe/structures/{structure_id}/ `solar_system_id` to `system_id`
-[] GET /v2/universe/factions/ `solar_system_id` to `system_id`
-[] GET /v1/universe/stars/{star_id}/ `solar_system_id` to `system_id`

**Comment by webba** [comment]
For I would suggest also 

- [ ] GET /v1/universe/races/ `alliance_id` to `faction_id`

**Comment by tsuthers-ccpgames** [comment]
related to above: https://github.com/ccpgames/esi-issues/issues/812
